src/main/kotlin/com/fantasy/ladbe/common/web/ErrorResponse.kt

```diff
@@ -0,0 +1,19 @@
+package com.fantasy.ladbe.common.web
+
+import java.time.LocalDateTime
+
+data class ErrorResponse(
+    var resultCode: String? = null,
+    var httpStatus: String? = null,
+    var httpMethod: String? = null,
```

Comment:
요 두 값들은 not-null 이어도 될것같은데, nullable 로 정하신 이유가 있어요?

Reply:
@philipjkim 
```suggestion
    val httpStatus: String = "",
    val httpMethod: String = "",
    val message: String = "",
```

어제 요한님과 회의 후 해당 부분이 위와 같이 수정되었습니다!